createdisk.sh

```diff
@@ -24,10 +24,10 @@ fi
 BASE_OS=${BASE_OS:-rhcos}
 OPENSHIFT_VERSION=$(${JQ} -r .clusterInfo.openshiftVersion $INSTALL_DIR/crc-bundle-info.json)
 
-# CRC_VM_NAME: short VM name to use in crc_libvirt.sh
+# SNC_PRODUCT_NAME: short VM name to use in crc_libvirt.sh
 # VM_PREFIX: full VM name with the random string generated by openshift-installer
-CRC_VM_NAME=${CRC_VM_NAME:-crc}
-VM_PREFIX=$(get_vm_prefix ${CRC_VM_NAME})
```

Comment:
"full VM name" for something called "VM_PREFIX" is so confusing :-/  It's is the short VM name + the random string generated by openshift-installer. Then full VM names/disk image names/... can be constructed by appending `-base`, `-master-0`, ... (I know you know that, just writing up explicitly what it is to help update the  comment ^^)